src/mbgl/tile/geometry_tile_data.cpp

```diff
@@ -36,6 +36,27 @@ static GeometryCollection toGeometryCollection(MultiPolygon<int16_t>&& multipoly
     return result;
 }
 
+// Attempts to restore original point order in rings:
+// see https://github.com/mapbox/mapbox-gl-native/issues/15268.
+static GeometryCollection restorePointOrder(GeometryCollection&& rings, const GeometryCollection& originalRings) {
```

Comment:
`GeometryCollection rings` and then just return, without move to avoid disabling NRVO